samwise.js

```diff
@@ -3,180 +3,168 @@
  *
```

Comment:
SCRIPT-Good job on updating the documentation along with the changes. It's very thorough and helpful!